src/Policeman/Diff.hs

```diff
@@ -30,15 +30,21 @@ comparePackageStructures psPrev psCur = PackageDiff {..}
     pdModule = setAddedRemoved (psModules psPrev) (psModules psCur)
 
     pdExport :: HashMap Module (Diff Export)
-    pdExport = foldl' go HM.empty oldModules
+    pdExport = foldl' go HM.empty oldExposedModules
       where
         go :: HashMap Module (Diff Export) -> (Module, ModuleStructure) -> HashMap Module (Diff Export)
         go hm (modul, msPrev) = case HM.lookup modul (psModulesMap psCur) of
             Just msCur -> HM.insert modul (moduleDiffExport msPrev msCur) hm
             Nothing    -> hm
 
         oldModules :: [(Module, ModuleStructure)]
-        oldModules = HM.toList $ psModulesMap psPrev
+        oldModules = HM.toList (psModulesMap psPrev)
+
+        wasExposed :: Module -> Bool
+        wasExposed x = x `member` psModules psCur
```

Comment:
It seems, that it's better to be called `isExposed` as it checks the current exposed list, not the previous version.

Reply:
oops! the name is right, I meant to check the previous version's exposed list, not the current exposed list.

Reply:
I pushed a fix. I believe this mistake addresses all three of your comments.